cmd/syncthing/gui.go

```diff
@@ -1151,6 +1145,11 @@ func (s *apiService) postDBScan(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+	nextStr := qs.Get("next")
+	next, err := strconv.Atoi(nextStr)
+	if err == nil {
+		s.model.DelayScan(folder, time.Duration(next)*time.Second)
```

Comment:
Right. Would it be better to add a function DelayScanAll or handle an empty folder string in the existing DelayScan function?